inst/doc/Processing_Controlled_Data_With_Bioconductor.md

```diff
@@ -181,6 +126,6 @@ docker ps # to get the container ID
 docker cp 1cf74ce69172:image_cel1.pdf .
 ```
 
-So, we mounted a bucket in a docker container. Read a raw PROTECTED CEL file from that
-bucket, and using a bioconductor package, produced an image of that microarray.
-And finally copied it out to our local system.
+So, gathered all the files, protected and open, associated with a barcode.
```

Comment:
So if the volume was mounted and if the image was written there, the order of events no longer matters.